css/style.css

```diff
@@ -0,0 +1,255 @@
+@import url('https://fonts.googleapis.com/css?family=Quicksand:300&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font: 12px/18px "San Francisco", "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 534px;
+  min-width: 1440px;
+  background-image: url("../images/your_verse_hero.jpg");
+  background-size: 1440px 1315px;
+  background-position-y: 534px;
+  text-align: center;
+}
+
+main {
+  display: flex;
+  margin-top: 2px;
+  width: 100%;
+  height: 200px;
+}
+
+footer {
+  width: 1015px;
+  height: 90px;
+  margin: 0 auto;
+  margin-top: 25px;
+}
+
+footer * {
+  color: #6e6e6e;
+}
+
+footer a {
+  font-size: 10px;
+  text-decoration: none;
+  padding: 0 4px;
+  color: #08c;
+}
+
+.header-text a {
+  color: white;
+  font-size: 25px;
+  text-decoration: none;
+}
+
+footer a:hover {
+  text-decoration: underline;
+}
+
+nav {
+  display: flex;
+  align-items: flex-end;
+  margin: 0 auto;
+  width: 980px;
+  height: 114px;
+}
+
+.container {
+  width: 1440px;
```

Comment:
#### It is better you set your container width to be set to 100%, that way it will be responsive to every desktop view. I suspect the inspection of the page gave you this value ```.1440px``` this is because that is the 100% value of the width of your screen. on a smaller desktop screen like mine, this will cause a horizontal scroll on the page.

Reply:
Thank you for your feedback. 
I didn't find a way to make everything come together nicely with percentages, and I resorted to using the width of the background image itself to be the baseline I would be using for every other width in the page. I agree with what you're saying, I thought auto margins would be enough to make everything fit just right regardless of viewport width, but it wasn't as good as I expected. I made sure it would still look right in common viewport sizes, like 1280 and 1920.